### PR TITLE
hubot-rocketchat: improve scripts/homer.coffee

### LINKED ISSUE #125: hubot-rocketchat: improve scripts/homer.coffee
* Translate the source code of `scripts/homer.coffee` from CoffeeScript into JavaScript (ES6).
* Rename the script into `scripts/the-simpsons-quotes.js`.
* Add the following quote to the 'try' quotes: "I can't promise I'll try, but I'll try to try.".
* Expand the list of types of food with `cake`, `pizza`, `doughnut`, `pancakes`, `hotdog`, `fries` and `hamburger`.

### PR DESCRIPTION
**Pull request by Aramgrigoryanwb**
Closes #125.

added new features for bot

### CODE REVIEW

File: hubot-rocketchat/scripts/the-simpsons-quotes.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+const beerQuotes = ["Beer... Now there's a temporary solution.", "All right, brain. You don't like me and I don't like you, but let's just do this and I can get back to killing you with beer.", "Mmmm... beer", "Beer: The cause of, and solution to, all of life's problems.", "Beer, beer, beer, bed, bed, bed.", "I would kill everyone in this room for a drop of sweet beer."]
+
+const tryQuotes = ["Kids, you tried your best and you failed miserably. The lesson is, never try.", "Trying is the first step towards failure.","I can't promise I'll try, but I'll try to try."]
+
+const internetQuotes = ["Oh, so they have internet on computers now!", "The Internet? Is that thing still around?"]
```
Comment (polina-popova) [inline hubot-rocketchat/scripts/the-simpsons-quotes.js]: Let's make it more readable as in the old version.

File: hubot-rocketchat/scripts/the-simpsons-quotes.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+
+module.exports = function(robot) {
+  robot.hear(/beer/i, function(msg) {
+    msg.send(msg.random(beerQuotes))
+  })
+  robot.hear(/bacon|bagel|barbecue|burger|candy|chocolate|donut|sandwich|breakfast|lunch|dinner|food|:cake:|:pizza:|:pancakes:|grub/i, function(msg) {
```
Comment (polina-popova) [inline hubot-rocketchat/scripts/the-simpsons-quotes.js]: Fix the regular expression to accept both emoji and written text.

Comment (eugulixes) [inline hubot-rocketchat/scripts/the-simpsons-quotes.js]: It's not necessary to keep `:`, i.e. do not write `:cake:`, `:pizza:`, etc.

File: hubot-rocketchat/scripts/the-simpsons-quotes.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+//    <food> - replies with "Mmmm... <food>"
+//    internet - replies with random internet quote
+//    try - replies with random try quote
+//  Author:
+//    bhankus
+
+const beerQuotes = [
+  "Beer... Now there's a temporary solution.",
+  "All right, brain. You don't like me and I don't like you, but let's just do this and I can get back to killing you with beer.",
+  "Mmmm... beer", "Beer: The cause of, and solution to, all of life's problems.",
```
Comment (eugulixes) [inline hubot-rocketchat/scripts/the-simpsons-quotes.js]: There are two quotes here. Let's split them.

File: hubot-rocketchat/scripts/the-simpsons-quotes.js [hunk 4]
```diff
@@ -0,0 +1,49 @@
+//  Description:
+//    Homer Simpson quotes.
```
Comment (eugulixes) [inline hubot-rocketchat/scripts/the-simpsons-quotes.js]: The quotes from an animated sitcom The Simpsons.

**Pull request merged**